fml/thread_unittests.cc

```diff
@@ -4,6 +4,11 @@
 
 #include "flutter/fml/thread.h"
 
+#if defined(OS_MACOSX) || defined(OS_LINUX) || defined(OS_ANDROID)
+#include <pthread.h>
+#endif
```

Comment:
#else errror "Some helpful text."

Reply:
If I add #error, whether it will cause tests on other platforms to fail?

Reply:
If this include is truly needed, on other platforms this will fail to compile anyways when pthread functions are used.  It's better to get an error message that explains the problem directly than getting a compilation error when it can't find the function.

Reply:
Oh, I'm just seeing you have more preprocessor guards below.  I would try to limit that to one preprocessor guard.  Not a huge deal though.

Reply:
At a bare minimum you can declare a variable that is reused between the 2 conditional compilations:
```c
#define FLUTTER_PTHREAD_SUPPORTED defined(OS_MACOSX) || defined(OS_LINUX) || defined(OS_ANDROID)
```

Reply:
Thanks for your advice. I had define a macro.